ingest/src/main/java/com/microsoft/azure/kusto/ingest/AzureStorageClient.java

```diff
@@ -3,6 +3,7 @@
 
 package com.microsoft.azure.kusto.ingest;
 
+import com.microsoft.azure.kusto.data.Ensure;
```

Comment:
Why?

(Also, not a fan of the Ensure class to begin with, as above)

Reply:
No reason for it to be on ingest anyway - data is common to both